apps/site/src/app/(home)/sections/FAQ/FAQ.module.scss

```diff
@@ -0,0 +1,13 @@
+.container {
+	background-image: url("~@/assets/backgrounds/faq-background-mobile.svg");
```

Comment:
Suggestion: considering it seems you need multiple height versions of this, it may be more appropriate to produce the background gradient natively with CSS. This will also conceptually reduce the amount of asset calls (but given how small the SVGs are, they may become inlined making this matter irrelevant).

Reply:
`fill: linear-gradient(180deg, #432810 0.32%, #2B1809 52.42%, #442911 100%);`